.gitignore

```diff
@@ -5,5 +5,4 @@ __pycache__
 .pytest_cache
 .tmp
 cookies.txt
-TLEData
```

Comment:
Review why this was removed. If a test needs a file in a certain location, the test should temporarily create it and delete it afterwards.

Reply:
Fixed

Reply:
So, what is the reason for keeping this in?

Reply:
Oh nevermind, I was misremembering the intent of my initial comment. Looks good.